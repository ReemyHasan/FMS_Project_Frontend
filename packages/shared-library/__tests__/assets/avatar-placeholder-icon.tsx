import React from "react";
import { render } from "@testing-library/react";
import TrsahIcon from "../../assets/icons/avatar-placeholder-icon";

describe("TrsahIcon", () => {
  it("renders correctly", () => {
    const { container } = render(<TrsahIcon />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it("renders with custom props", () => {
    const { container } = render(<TrsahIcon width={20} height={20} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
