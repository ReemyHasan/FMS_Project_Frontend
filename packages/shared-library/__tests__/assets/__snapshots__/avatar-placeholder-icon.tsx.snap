// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrsahIcon renders correctly 1`] = `
<span
  class="anticon"
  role="img"
>
  <svg
    fill="none"
    viewBox="0 0 64 64"
    xmlns="http://www.w3.org/2000/svg"
  >
    <rect
      fill="#F2F4F7"
      height="64"
      rx="32"
      width="64"
    />
    <path
      d="M42.6666 44C42.6666 42.1392 42.6666 41.2089 42.4369 40.4518C41.9199 38.7473 40.586 37.4134 38.8814 36.8963C38.1244 36.6667 37.194 36.6667 35.3332 36.6667H28.6666C26.8058 36.6667 25.8755 36.6667 25.1184 36.8963C23.4139 37.4134 22.08 38.7473 21.5629 40.4518C21.3333 41.2089 21.3333 42.1392 21.3333 44M37.9999 26C37.9999 29.3137 35.3136 32 31.9999 32C28.6862 32 25.9999 29.3137 25.9999 26C25.9999 22.6863 28.6862 20 31.9999 20C35.3136 20 37.9999 22.6863 37.9999 26Z"
      stroke="#475467"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2.66667"
    />
  </svg>
</span>
`;

exports[`TrsahIcon renders with custom props 1`] = `
<span
  class="anticon"
  height="20"
  role="img"
  width="20"
>
  <svg
    fill="none"
    height="20"
    viewBox="0 0 64 64"
    width="20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <rect
      fill="#F2F4F7"
      height="64"
      rx="32"
      width="64"
    />
    <path
      d="M42.6666 44C42.6666 42.1392 42.6666 41.2089 42.4369 40.4518C41.9199 38.7473 40.586 37.4134 38.8814 36.8963C38.1244 36.6667 37.194 36.6667 35.3332 36.6667H28.6666C26.8058 36.6667 25.8755 36.6667 25.1184 36.8963C23.4139 37.4134 22.08 38.7473 21.5629 40.4518C21.3333 41.2089 21.3333 42.1392 21.3333 44M37.9999 26C37.9999 29.3137 35.3136 32 31.9999 32C28.6862 32 25.9999 29.3137 25.9999 26C25.9999 22.6863 28.6862 20 31.9999 20C35.3136 20 37.9999 22.6863 37.9999 26Z"
      stroke="#475467"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2.66667"
    />
  </svg>
</span>
`;
